template/src/project_name/accounts/tests.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+
+class UsersManagersTests(TestCase):
+
+    def test_create_user(self):
+        User = get_user_model()
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** Assert that it return the correct class

If the settings are incorrect, then `get_user_model` will return the standard, default user model. Since our custom model is functionally exactly the same as the default one - this test would still pass.‏

It would be a good idea to create a separate test to ensure it actually returns the expected class.